sample/calc.y

```diff
@@ -0,0 +1,97 @@
+/*
+ * How to build and run:
+ *
+ * $ lrama -d calc.y -o calc.c && gcc -Wall calc.c -o calc && ./calc
+ * 1
+ * => 1
+ * 1+2*3
+ * => 7
+ * (1+2)*3
+ * => 9
+ *
+ */
+
+%{
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+
+#include "calc.h"
```

Comment:
[note] Bison automatically insert `#include "calc.h"`, but lrama does not. I will revisit this point later.